tests/components/vizio/test_init.py

```diff
@@ -37,7 +38,12 @@ async def test_load_and_unload(
     assert await hass.config_entries.async_setup(config_entry.entry_id)
     await hass.async_block_till_done()
     assert len(hass.states.async_entity_ids(MP_DOMAIN)) == 1
+    assert DOMAIN in hass.data
+    assert "apps" in hass.data[DOMAIN]
+    assert isinstance(hass.data[DOMAIN]["apps"], DataUpdateCoordinator)
```

Comment:
We shouldn't assert integration details. Limit this to check for `DOMAIN` in hass.data`.

Reply:
https://github.com/home-assistant/core/pull/39579